src/utils/DEMUtils.js

```diff
@@ -371,7 +371,7 @@ function _readZ(layer, method, coord, nodes, cache) {
     }
 
     const tile = tileWithValidElevationTexture;
-    const src = tileWithValidElevationTexture.getLayerTextures(l_ELEVATION)[0];
+    const src = tileWithValidElevationTexture.material.textures[l_ELEVATION][0];
```

Comment:
`tileWithValidElevationTexture.getLayerTextures(layer)[0]` ?

Reply:
Yep, thanks.

Reply:
Ah no, because `layer` here is the geometry layer. And the calling site is not aware of which elevation layer we can use.

So for now I'll leave it like that.

Reply:
and with the famous `tile.layer`?
you could write:
`tileWithValidElevationTexture.getLayerTextures(tile.layer)[0]`

Reply:
> and with the famous tile.layer?

Are you sure the `famous` bit is needed?

> you could write: tileWithValidElevationTexture.getLayerTextures(tile.layer)[0]

And it would be incorrect. `tile.layer` is the GeometryLayer, not the Layer used for the elevation.

Reply:
Ok,
So we don't know in elevation layer the data is searched for?

Reply:
we should add tests to DEMUtils, if it doesn't exist

Reply:
Not the scope of this PR. I'm only changing this file to remove one useless abstraction (and it's a 2 line diff)